src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -0,0 +1,966 @@
+                if (centralFanTypeNum == DataHVACGlobals::FanType_SystemModelObject) {
+                    // 2022-02-04: This type is zero indexed.
+                    // 2022-02-04: Need to process the System fan here first
+                    state.dataHVACFan->fanObjs.emplace_back(new HVACFan::FanSystem(state, centralFanName));
+
+                    centralFanIndex = HVACFan::getFanObjectVectorIndex(state, centralFanName); // zero-based
+                    if (centralFanIndex >= 0) {
+                        availSchNum = state.dataHVACFan->fanObjs[centralFanIndex]->availSchedIndex;
+                        // normal index
+                        SetupOutputVariable(state,
+                                            "Central Exhaust Fan Outlet Air Mass Flow Rate",
+                                            OutputProcessor::Unit::kg_s,
+                                            thisExhSys.centralFan_MassFlowRate,
+                                            OutputProcessor::SOVTimeStepType::System,
+                                            OutputProcessor::SOVStoreType::Average,
+                                            thisExhSys.Name);
+
+                        SetupOutputVariable(state,
+                                            "Central Exhaust Fan Outlet Air Volume Flow Rate",
+                                            OutputProcessor::Unit::m3_s,
+                                            thisExhSys.centralFan_VolumeFlowRate,
+                                            OutputProcessor::SOVTimeStepType::System,
+                                            OutputProcessor::SOVStoreType::Average,
+                                            thisExhSys.Name);
+
+                        SetupOutputVariable(state,
+                                            "Central Exhaust Fan Power",
+                                            OutputProcessor::Unit::W,
+                                            thisExhSys.centralFan_Power,
+                                            OutputProcessor::SOVTimeStepType::System,
+                                            OutputProcessor::SOVStoreType::Average,
+                                            thisExhSys.Name);
+
+                        SetupOutputVariable(state,
+                                            "Central Exhaust Fan Energy",
+                                            OutputProcessor::Unit::J,
+                                            thisExhSys.centralFan_Energy,
+                                            OutputProcessor::SOVTimeStepType::System,
+                                            OutputProcessor::SOVStoreType::Summed,
+                                            thisExhSys.Name);
+
+                    } else {
+                        centralFanIndex = -1;
+                        // here a severe error message is needed
+                        ShowSevereError(state, RoutineName + cCurrentModuleObject + "=" + thisExhSys.Name);
```

Comment:
Use `format` rather than string concatenation.

Reply:
This is now done for all similar occurrences in the file.